docs/en/operations/server-configuration-parameters/settings.md

```diff
@@ -5,6 +5,30 @@ sidebar_label: Server Settings
 
 # Server Settings {#server-settings}
 
+## allow_nondeterministic_mutations {#allow_nondeterministic_mutations}
+
```

Comment:
It's a query setting, not a server configuration parameter. Please move it to `docs/en/operations/settings/settings.md`.

Reply:
moved!